docs/architecture.md

```diff
@@ -82,3 +82,9 @@ graph LR;
     click passenger_events "../tables/#passenger-events"
     click fare_transactions "../tables/#fare-transactions"
 ```
+
+!!! tip "Relationship of `trip_id_scheduled` to GTFS Schedule `trip_id`"
+
+    `trip_id_scheduled` in TIDES and `trip_id` in GTFS Schedule should be consistent when a performed trip exists in the published GTFS Schedule. 
+
+    If a trip does not exist in the published GTFS Schedule, `trip_id` values in TIDES do not have any assumed relationship. 
```

Comment:
Do you mean `trip_id_scheduled` values in TIDES do not have any assumed relationship?  And does that mean that they are blank/null?

Separately, if an agency is using scheudule data not in GTFS format, `trip_id_scheduled` might reference that.  So should we say "If a trip does not exist in the published GTFS (or other) schedule, ..."

Reply:
Thanks for catching! I think that since this refers to GTFS specifically that is a bit moot. I added a clarification that " any relationship...with GTFS"